alfasim_sdk/cli.py

```diff
@@ -61,7 +61,7 @@ def template(dst, caption, plugin_id, author_name, author_email):
     The template folder will be placed on the ``dst`` option, that by default is the current directory from where the command
```

Comment:
We need to change this docstring into a raw string to fix the build, change line 58 from `"""` to `r"""`